### PR TITLE
Make the sponsorship tiers consistent

### LINKED ISSUE #49: Improve sponsor page
* make top tier 1000 eur
* update difference between small and medium
* unify theme (metal or size?)

### PR DESCRIPTION
**Pull request by endocrimes**
closes #49

### CODE REVIEW

**Comment by vsmart** [comment]
Thanks @DanToml 🎉 
Do you want to go ahead and fix the other 2 items from https://github.com/clojurebridge-berlin/clojurebridge-berlin.github.io/issues/49 too? If not, I'm happy to merge this as is.

**Comment by endocrimes** [comment]
@vsmart I've updated the platinum tier too now, not sure how we were going to differentiate the silver and gold tiers tho.

**Comment by vsmart** [comment]
@DanToml Oh yea. I think we said to remove "Your logo on our website" from the bottom tier? @daveliepmann What do you think?

**Comment by daveliepmann** [comment]
Thanks for the ping! My recollection is that there was rough consensus to 

 1. increase platinum from 600 to 1000€ or 2500€
 2. remove "logo on website" from lowest tier
 
I also humbly suggest removing the "e.g." lines. They're supporting the event, not paying for coffee or t-shirts (especially if we don't have t-shirts). If we really want don't want to remove it then I further and even more humbly suggest rewording to something like "this would cover our costs for <expense>".

**Comment by endocrimes** [comment]
@daveliepmann agreed, and updated pr

**Comment by daveliepmann** [comment]
Looks good to me.

**Comment by vsmart** [comment]
@daveliepmann @DanToml The reason we have those examples in there is because we have sometimes had sponsors pay directly for items like t-shirts or catering, rather than giving us money first. This helps us because we have less billing hassle, and less sending-money-back and forth over the Travis Foundation. I've also had companies question what we even need money for since it's a free event. I'd quite like to keep the examples, but I'm happy to reword to "this could cover ..."

**Comment by daveliepmann** [comment]
Sounds like the rewording would be best.

**Comment by endocrimes** [comment]
Rewording would be great, I'll remove that commit from this pr and do a follow up.

**Comment by vsmart** [comment]
Yay! Thanks @DanToml! 

high five
